templates/inspec/singular_resource.erb

```diff
@@ -20,7 +20,7 @@
 
   inside_indent = 8
 
-  requires = generate_requires(object.all_user_properties)
+  requires = generate_requires(object.properties)
```

Comment:
Why is this just properties?

Reply:
Fixed this so that we generate the nested object files for all properties

Reply:
I don't follow. Why does a lack of parameters make that possible?

Reply:
I mean, I reverted this and made it so that we generate nested objects for parameters as well. 

This is required because compute disk takes an encryption key as a parameter, which was previously not being generated